selfdrive/car/tests/test_docs.py

```diff
@@ -74,8 +74,10 @@ def test_harnesses(self):
         if car.name == "comma body":
           raise unittest.SkipTest
 
-        self.assertNotIn(car.harness_kit.connector, [None, Harness.none], f"Need to specify car harness: {car.name}")
-        self.assertTrue(car.harness_kit.parts, f"Need to specify harness parts: {car.name} with {car.harness_kit.connector.value} connector")
+        self.assertTrue(len(car.car_parts.parts) > 0, f"Need to specify car parts: {car.name}")
+        self.assertTrue(len(list(filter((lambda p: p.value.type is PartType.connector), car.car_parts.parts))) == 1, f"Need to specify exactly ONE harness connector: {car.name}")
+        self.assertTrue(len(list(filter((lambda p: p.value.type is PartType.mount), car.car_parts.parts))) == 1, f"Need to specify exactly ONE mount: {car.name}")
+        self.assertTrue(len(list(filter((lambda p: p is CarPart.right_angle_obd_c_cable_1_5ft), car.car_parts.parts))) > 0, f"Need to specify a right angle OBD-C cable (1.5ft): {car.name}")
```

Comment:
```suggestion
        car_part_types = [p.value.type for p in car.car_parts.parts]
        self.assertTrue(len(car.car_parts.parts) > 0, f"Need to specify car parts: {car.name}")
        self.assertTrue(car_part_types.count(PartType.connector), f"Need to specify exactly ONE harness connector: {car.name}")
        self.assertTrue(car_part_types.count(PartType.mount) == 1, f"Need to specify exactly ONE mount: {car.name}")
        self.assertTrue(CarPart.right_angle_obd_c_cable_1_5ft in car.car_parts.parts, f"Need to specify a right angle OBD-C cable (1.5ft): {car.name}")
```

this is a bit cleaner